src/frame/js/actions/faye.js

```diff
@@ -161,10 +163,11 @@ function updateUser(currentAppUser, nextAppUser, client) {
                 setItem(`${appId}.sessionToken`, nextAppUser.sessionToken);
             }
 
+            dispatch(disconnectFaye());
```

Comment:
No because `disconnectFaye` is a thunk and `batchActions` expects raw action objects